installers/metrics/ansible/roles/pgo-metrics/tasks/main.yml

```diff
@@ -79,6 +79,13 @@
     - include_tasks: grafana.yml
       when: grafana_install | bool
 
+    - name: Check if Timeout Flag Supported
+      command: |
+        {{ kubectl_or_oc }} rollout status deployment/flagcheck -n {{ metrics_namespace }} \
+          --watch=false --timeout=1s
```

Comment:
Instead of actually hitting the API, can we use the `--help` flag? On recent versions of `kubectl` and `oc` it goes to stdout:

```
$ kubectl rollout status --help | wc -l
20
```

Reply:
Good thought, the task has been updated to use `--help`.